integration-cli/docker_cli_start_volume_driver_unix_test.go

```diff
@@ -362,3 +420,24 @@ func (s *DockerExternalVolumeSuite) TestStartExternalVolumeDriverBindExternalVol
 	c.Assert(mounts[0].Name, check.Equals, "foo")
 	c.Assert(mounts[0].Driver, check.Equals, "test-external-volume-driver")
 }
+
+func (s *DockerExternalVolumeSuite) TestStartExternalVolumeDriverList(c *check.C) {
+	dockerCmd(c, "volume", "create", "-d", "test-external-volume-driver", "--name", "abc")
+	out, _ := dockerCmd(c, "volume", "ls")
+	ls := strings.Split(strings.TrimSpace(out), "\n")
+	c.Assert(len(ls), check.Equals, 2, check.Commentf("\n%s", out))
+
+	vol := strings.Fields(ls[len(ls)-1])
+	c.Assert(len(vol), check.Equals, 2, check.Commentf("%v", vol))
+	c.Assert(vol[0], check.Equals, "test-external-volume-driver")
```

Comment:
This is not 2 volumes, it is one volume and we're checking the driver name and name are correct.